sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -734,10 +760,23 @@ private boolean isRollingSessionReceiver() {
         private ReceiveMode receiveMode = ReceiveMode.PEEK_LOCK;
         private String subscriptionName;
         private String topicName;
+        private Duration maxAutoLockRenewalDuration;
 
         private ServiceBusReceiverClientBuilder() {
         }
 
+        /**
+         * Enables auto-lock renewal by renewing session lock renewal until the {@code maxAutoLockRenewalDuration} has
+         * elapsed.
+         *
+         * @param maxAutoLockRenewalDuration Maximum amount of time to renew the session lock.
+         * @return The modified {@link ServiceBusReceiverClientBuilder} object.
```

Comment:
This documentation should be for non session receiver because this builder belongs to non-session entities.